www/grpc/transaction_test.go

```diff
@@ -105,3 +106,70 @@ func TestSendRawTransaction(t *testing.T) {
 	})
 	assert.Nil(t, conn.Close(), "Error closing connection")
 }
+
+func TestGetRawTransaction(t *testing.T) {
+	ts := testsuite.NewTestSuite(t)
+
+	conn, client := testTransactionClient(t)
+
+	t.Run("Transfer", func(t *testing.T) {
+		tx, privateKey := ts.GenerateTestTransferTx()
+
+		res, err := client.GetRawTransferTransaction(tCtx, &pactus.GetRawTransferTransactionRequest{
+			LockTime: tx.LockTime(),
+			Sender:   privateKey.PublicKeyNative().AccountAddress().String(),
+			Receiver: tx.Payload().Receiver().String(),
+			Amount:   tx.Payload().Value(),
+			Fee:      tx.Fee(),
+			Memo:     tx.Memo(),
+		})
+		assert.Nil(t, err)
+		fmt.Println(res.RawTransaction)
+	})
+
+	t.Run("Bond", func(t *testing.T) {
+		tx, privateKey := ts.GenerateTestBondTx()
+
+		res, err := client.GetRawBondTransaction(tCtx, &pactus.GetRawBondTransactionRequest{
+			LockTime:  tx.LockTime(),
+			Sender:    privateKey.PublicKeyNative().AccountAddress().String(),
+			Receiver:  tx.Payload().Receiver().String(),
+			Stake:     tx.Payload().Value(),
+			PublicKey: "",
+			Fee:       tx.Fee(),
+			Memo:      tx.Memo(),
+		})
+		assert.Nil(t, err)
+		fmt.Println(res.RawTransaction)
+	})
+
+	t.Run("UnBond", func(t *testing.T) {
+		valAddr := ts.RandValAddress()
```

Comment:
To be compatible with the rest of the test: use: `tx, privateKey := ts.GenerateTestUnbondTx()`